package.sh

```diff
@@ -104,44 +127,46 @@ if [[ $? -ne 0 ]]; then
 fi
 
 if [[ -n "${CI-}" ]] && [[ "$PLATFORM" == "darwin" ]] && [[ -n "${CERTIFICATE_OSX_APPLICATION-}" ]]; then
-  echo "Preparing to sign backend binary for macos"
-  KEY_CHAIN=rotki-build.keychain
-  CSC_LINK=/tmp/certificate.p12
-  export CSC_LINK
-  # Recreate the certificate from the secure environment variable
-  echo $CERTIFICATE_OSX_APPLICATION | base64 --decode > $CSC_LINK
-  #create a keychain
-  security create-keychain -p actions $KEY_CHAIN
-  # Make the keychain the default so identities are found
-  security default-keychain -s $KEY_CHAIN
-  # Unlock the keychains
-  security unlock-keychain -p actions $KEY_CHAIN
-  security import $CSC_LINK -k $KEY_CHAIN -P $CSC_KEY_PASSWORD -T /usr/bin/codesign;
-  security set-key-partition-list -S apple-tool:,apple: -s -k actions $KEY_CHAIN
-
-  echo "::group::Preparing to sign"
-  files=(`find ./${BACKEND_DIST_DIR} -type f -exec ls -dl \{\} \; | awk '{ print $9 }'`)
-  for i in "${files[@]}"
-  do
-    echo "Signing $i"
-    codesign --force --options runtime --entitlements ./packaging/entitlements.plist --sign $IDENTITY $i --timestamp || exit 1
-    codesign --verify --verbose $i || exit 1
-  done
-  echo "::endgroup::"
+    echo "Preparing to sign backend binary for macos"
+    KEY_CHAIN=rotki-build.keychain
+    CSC_LINK=/tmp/certificate.p12
```

Comment:
Should this be a variable?

Reply:
I don't see why.

This is a temporary file created by the `CERTIFICATE_OSX_APPLICATION` env variable